velox/exec/tests/OrderByTest.cpp

```diff
@@ -1033,7 +1033,7 @@ DEBUG_ONLY_TEST_F(OrderByTest, reclaimDuringOutputProcessing) {
       ASSERT_GT(reclaimableBytes, 0);
       reclaimerStats_.reset();
       reclaimAndRestoreCapacity(op, reclaimableBytes, reclaimerStats_);
-      ASSERT_EQ(reclaimerStats_.reclaimedBytes, reclaimableBytes);
+      ASSERT_LE(reclaimerStats_.reclaimedBytes, reclaimableBytes);
```

Comment:
Curious, why we need to change the check condition in this PR? Thanks!

Reply:
Because of reservation quantization the freed used memory might not be reflected to reserved memory.